packages/@okta/vuepress-theme-prose/components/Footer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <footer class="app-footer">
-    <p>
+    <p class="app-footer--need-support">
```

Comment:
Noticed that the footer on the homepage lost the styling on the text here so this is a tiny fix

Reply:
**now**:
![Screen Shot 2021-05-17 at 3 33 43 PM](https://user-images.githubusercontent.com/72248639/118545945-49841d00-b725-11eb-9609-437ff0c82652.png)

**fixed**:
![Screen Shot 2021-05-17 at 3 33 47 PM](https://user-images.githubusercontent.com/72248639/118545947-4ab54a00-b725-11eb-81ef-cfb35b90f756.png)